.github/workflows/format-lint.yml

```diff
@@ -0,0 +1,57 @@
+name: python3 setup.py lint
+
+# Run CI only when a release is created, on changes to main branch, or any PR
+# to main. Do not run CI on any other branch. Also, skip any non-source changes
+# from running on CI
+on:
+  release:
+    types: [created]
+  push:
+    branches: main
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '.gitignore'
+      - 'README.rst'
+      - '.github/workflows/build.yml'
+      - '.github/workflows/manylinux.yml'
+      - '.github/workflows/ubuntu-sdist.yml'
+
+  pull_request:
+    branches: main
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '.gitignore'
+      - 'README.rst'
+      - '.github/workflows/build.yml'
+      - '.github/workflows/manylinux.yml'
+      - '.github/workflows/ubuntu-sdist.yml'
+
+jobs:
+  format-lint-code-check:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Install deps
+        run: |
+          sudo apt-get update --fix-missing
+          sudo apt-get upgrade
+          sudo apt-get install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev \
+            libfreetype6-dev libportmidi-dev libjpeg8-dev python3-setuptools \
+            python3-dev python3-numpy libsdl2-ttf-dev
```

Comment:
we don't need any of these apt lines here because we don't build anything here :)

Reply:
Indeed. I will remove them.

Reply:
Runs lots faster without them!